src/migrations/1661850289882-callout-notification-preferences.ts

```diff
@@ -0,0 +1,41 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import {
+  addPreferenceDefinitions,
+  addPreferencesToUsers,
+  DefinitionInsertType,
+  PreferenceInsertType,
+  removePreferences,
+} from '@src/migrations/utils/preferences';
+import { UserPreferenceType } from '@src/common';
+import { PreferenceType } from '@common/enums/preference.type';
+
+export class calloutNotificationPreferences1661850289882
+  implements MigrationInterface
+{
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    const definitions: DefinitionInsertType[] = [
+      {
+        definitionSet: 'user',
+        group: 'Notification',
+        displayName: 'New callout created',
+        description:
+          'Receive a notification when a callout is created in a community I am a member of',
```

Comment:
Receive a notification when a Callout is published in a community I am a member of